### ISSUE TITLE
[0.8.4] Instantiate mutable configuration files fails as files are not found (in 19.09)

### ISSUE THREAD

**Issue by pcm32** [galaxyproject/ansible-galaxy#93]
When running 0.8.4 version of this role (as on 0.9.0 I don't get that far), in the mutable setup, it fails on:

```ansible
TASK [galaxyproject.galaxy : Instantiate mutable configuration files] ******************************************************************************************************************************
failed: [hx-noah-01-06] (item={u'dest': u'/some-path/galaxy-gxa-001-data/config/shed_tool_conf.xml', u'src': u'/some-path/galaxy-19.09/config/shed_tool_conf.xml.sample'}) => {"changed": true, "cmd": ["cp", "/some-path/galaxy-19.09/config/shed_tool_conf.xml.sample", "/some-path/galaxy-gxa-001-data/config/shed_tool_conf.xml"], "delta": "0:00:00.003480", "end": "2019-10-28 11:42:48.868639", "item": {"dest": "/some-path/galaxy-gxa-001-data/config/shed_tool_conf.xml", "src": "/some-path/galaxy-19.09/config/shed_tool_conf.xml.sample"}, "msg": "non-zero return code", "rc": 1, "start": "2019-10-28 11:42:48.865159", "stderr": "cp: cannot stat ‘/some-path/galaxy-19.09/config/shed_tool_conf.xml.sample’: No such file or directory", "stderr_lines": ["cp: cannot stat ‘/some-path/galaxy-19.09/config/shed_tool_conf.xml.sample’: No such file or directory"], "stdout": "", "stdout_lines": []}
failed: [hx-noah-01-06] (item={u'dest': u'/some-path/galaxy-gxa-001-data/config/migrated_tools_conf.xml', u'src': u'/some-path/galaxy-19.09/config/migrated_tools_conf.xml.sample'}) => {"changed": true, "cmd": ["cp", "/some-path/galaxy-19.09/config/migrated_tools_conf.xml.sample", "/some-path/galaxy-gxa-001-data/config/migrated_tools_conf.xml"], "delta": "0:00:00.002804", "end": "2019-10-28 11:42:51.011456", "item": {"dest": "/some-path/galaxy-gxa-001-data/config/migrated_tools_conf.xml", "src": "/some-path/galaxy-19.09/config/migrated_tools_conf.xml.sample"}, "msg": "non-zero return code", "rc": 1, "start": "2019-10-28 11:42:51.008652", "stderr": "cp: cannot stat ‘/some-path/galaxy-19.09/config/migrated_tools_conf.xml.sample’: No such file or directory", "stderr_lines": ["cp: cannot stat ‘/some-path/galaxy-19.09/config/migrated_tools_conf.xml.sample’: No such file or directory"], "stdout": "", "stdout_lines": []}
failed: [hx-noah-01-06] (item={u'dest': u'/some-path/galaxy-gxa-001-data/config/shed_data_manager_conf.xml', u'src': u'/some-path/galaxy-19.09/config/shed_data_manager_conf.xml.sample'}) => {"changed": true, "cmd": ["cp", "/some-path/galaxy-19.09/config/shed_data_manager_conf.xml.sample", "/some-path/galaxy-gxa-001-data/config/shed_data_manager_conf.xml"], "delta": "0:00:00.002556", "end": "2019-10-28 11:42:53.245771", "item": {"dest": "/some-path/galaxy-gxa-001-data/config/shed_data_manager_conf.xml", "src": "/some-path/galaxy-19.09/config/shed_data_manager_conf.xml.sample"}, "msg": "non-zero return code", "rc": 1, "start": "2019-10-28 11:42:53.243215", "stderr": "cp: cannot stat ‘/some-path/galaxy-19.09/config/shed_data_manager_conf.xml.sample’: No such file or directory", "stderr_lines": ["cp: cannot stat ‘/some-path/galaxy-19.09/config/shed_data_manager_conf.xml.sample’: No such file or directory"], "stdout": "", "stdout_lines": []}
failed: [hx-noah-01-06] (item={u'dest': u'/some-path/galaxy-gxa-001-data/config/shed_tool_data_table_conf.xml', u'src': u'/some-path/galaxy-19.09/config/shed_tool_data_table_conf.xml.sample'}) => {"changed": true, "cmd": ["cp", "/some-path/galaxy-19.09/config/shed_tool_data_table_conf.xml.sample", "/some-path/galaxy-gxa-001-data/config/shed_tool_data_table_conf.xml"], "delta": "0:00:00.002730", "end": "2019-10-28 11:42:55.385746", "item": {"dest": "/some-path/galaxy-gxa-001-data/config/shed_tool_data_table_conf.xml", "src": "/some-path/galaxy-19.09/config/shed_tool_data_table_conf.xml.sample"}, "msg": "non-zero return code", "rc": 1, "start": "2019-10-28 11:42:55.383016", "stderr": "cp: cannot stat ‘/some-path/galaxy-19.09/config/shed_tool_data_table_conf.xml.sample’: No such file or directory", "stderr_lines": ["cp: cannot stat ‘/some-path/galaxy-19.09/config/shed_tool_data_table_conf.xml.sample’: No such file or directory"], "stdout": "", "stdout_lines": []}
	to retry, use: --limit @/Users/pmoreno/Development/ansible/galaxy-lsf/playbook/playbook.retry
```

as none of the files listed in `galaxy_mutable_configs` seem to be there anymore (the samples) on 19.09 checkout.

**Comment by pcm32** [comment]
is 19.09 supported by this role?

**Comment by pcm32** [comment]
I can set the variable `galaxy_mutable_configs` to an empty hash, but then the issue I get is that on run.sh the file `shed_tool_conf.xml` is not found.

**Comment by natefoo** [comment]
This is fixed in 0.9.0, so I don't think there's anything to do here other than get 0.9.0 working for you (I'll follow up in #92).
